src/github/handlers/help-command.ts

```diff
@@ -27,12 +27,16 @@ export async function postHelpCommand(context: GitHubContext<"issue_comment.crea
     const { plugin } = pluginElement.uses[0];
     commands.push(...(await parseCommandsFromManifest(context, plugin)));
   }
-  await context.octokit.rest.issues.createComment({
-    body: comments.concat(commands.sort()).join("\n"),
-    issue_number: context.payload.issue.number,
-    owner: context.payload.repository.owner.login,
-    repo: context.payload.repository.name,
-  });
+  if (!commands.length) {
+    console.warn("No commands found, will not post the help command message.");
```

Comment:
I thought we use our logger instead?

Reply:
It's the kernel so our logger instance does not exist here, but yes ideally we should use it.